bundle/manifests/nvidia.com_clusterpolicies.yaml

```diff
@@ -1523,14 +1523,6 @@ spec:
                       set by external tools to store and retrieve arbitrary metadata. They are not
                       queryable and should be preserved when modifying objects.
                     type: object
-                  defaultRuntime:
```

Comment:
@meskinemohamed We can't remove this as it is a change in the CRD's API spec. This will break the deployments of users